Lib/test/test_email/test_email.py

```diff
@@ -3164,6 +3164,14 @@ def test_escape_backslashes(self):
         b = 'person@dom.ain'
         self.assertEqual(utils.parseaddr(utils.formataddr((a, b))), (a, b))
 
+    def test_parseaddr_formataddr_inverse(self):
+        # gh-91400
+        identity = '"foo  bar" <foo@example.com>'
+        single = utils.formataddr(utils.parseaddr(identity))
+        double = utils.formataddr(utils.parseaddr(single))
```

Comment:
Do we really need to repeat the check twice? It doesn't make sense to me. If the first line returns the string unchanged, we have `identity == single` anyway, and so the second line gets the same string as the first line.

Reply:
Before the change in `email/utils.py`, this unit tests fails. Although your comment reflects what would be expected. See the issue (https://github.com/python/cpython/issues/91400) for example.

Reply:
My point is that if it doesn't change on the first try, it won't change on the second try either. The second call is just unnecessary and confusing. In the issue it changes on the first call, and that's what the test should check.